src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/DependencyAnalysis/PInvokeMethodFixupNode.cs

```diff
@@ -113,6 +121,7 @@ public override int CompareToImpl(ISortableNode other, CompilerComparer comparer
         public readonly PInvokeModuleData ModuleData;
         public readonly string EntryPointName;
         public readonly CharSet CharSetMangling;
+        public readonly int SignatureBytes;
```

Comment:
This should be included in PInvokeMethodData.Equals.

Reply:
Done